### PR TITLE
Js modules

### PR DESCRIPTION
**Pull request by aizjicod**
on this pull request, I did the following

- change the js files in order to use modules for better and cleaner code.
- keep the functionality of the page.
- change the time in order to use a library called lunox.

download

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable max-classes-per-file */
```
Comment (mikethreels) [inline index.js]: - [ ] please avoid disabling any linters, linters are used to improve your code and make it more readable.

File: modules/books-class.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/* eslint-disable max-classes-per-file */
```
Comment (mikethreels) [inline modules/books-class.js]: - [ ] please avoid disabling any linters, linters are used to improve your code and make it more readable.

File: modules/validations.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-classes-per-file */
```
Comment (mikethreels) [inline modules/validations.js]: - [ ] please avoid disabling any linters, linters are used to improve your code and make it more readable.

File: modules/validations.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const showmessage = (test, msg) => {
+  if (test) {
+    error.textContent = msg;
+    return test;
+  }
+  error.textContent = msg;
+  return test;
+};
+
+export default function validations(input, msg, input2, msg2) {
```
Comment (mikethreels) [inline modules/validations.js]: - [ ] please make use of ES6 which included arrow functions instead of the function keyword.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Chino, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**